contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +114,67 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+== Anchoring titles and section headings
+
+All titles and section headings must have an anchor. The anchor ID must be similar to the title or section heading.
+
+=== Anchoring in assembly files
+
+The following is an example anchor in an assembly file:
+
+----
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+----
+
+[NOTE]
+====
+Do not include line spaces between the anchor and the section title.
+====
+
+=== Anchoring in module files
+
+You must add the `{context}` variable to the end of anchor IDs in module files. When called, the `{context}` variable is resolved into the value declared in the `:context:` attribute in the corresponding assembly file. This enables cross-referencing to module IDs in context to a specific assembly and is useful when a module is included in multiple assemblies.
+
+The following is an example of an anchor in a module file:
+
+----
+[id="default-monitoring-components_{context}"]
+= Default monitoring components
+----
+
+[NOTE]
+====
+The `{context}` variable must be preceded by an underscore (`_`) when declared in an anchor ID.
+====
+
+=== Creating unique anchor IDs for "Prerequisites", "Additional resources", and "Next steps" titles
```

Comment:
In this section, we are specifically talking about these in the context of assemblies and not modules, although just by reading the title of this section my assumption would be that we are talking about these in any context. Not sure if we need to say something about modules as well?

Reply:
I have updated the title to the following, to make it clear that the section is specific to assemblies. I restructured the sentence to minimise it and still convey the full meaning:

`=== Anchoring "Prerequisites", "Additional resources", and "Next steps" titles in assemblies`